extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -402,7 +407,7 @@ export class JupyterServerInstallation {
 		let installJupyterCommand: string;
 		if (process.platform === constants.winPlatform) {
 			let requirements = path.join(this._pythonPackageDir, 'requirements.txt');
-			installJupyterCommand = `"${this._pythonExecutable}" -m pip install --no-index -r "${requirements}" --find-links "${this._pythonPackageDir}" --no-warn-script-location`;
+			installJupyterCommand = `"${this._pythonExecutable}" -m pip install --user --no-index -r "${requirements}" --find-links "${this._pythonPackageDir}" --no-warn-script-location`;
```

Comment:
there aren't any adverse effects to using --user, right?

Reply:
So far I haven't seen any issues with it in local testing or the integration tests. We include the user package directory in our path in addition to the normal python paths, so it can fallback to those if needed.